tests/test_lab_view.py

```diff
@@ -91,7 +93,7 @@ def test_request_resources(self):
 
             self.assertEqual(
                 "LOCKED",
-                self.sample_view.get_sample_position_status("furnace_1/inside")[0].name,
+                self.sample_view.get_sample_position_status("furnace_1/inside/1")[0].name,
```

Comment:
could you explain me this change?

Reply:
Because now the `furnace_1/inside` has more sample positions. According to the logic defined in creating sample positions, the name is now `furnace_1/inside/1`.

Reply:
added